From 5abf157e569356fc7234818d8009fe34f0da6704 Mon Sep 17 00:00:00 2001
From: Jung Zhao <jung.zhao@rock-chips.com>
Date: Thu, 18 Jan 2018 15:39:45 +0800
Subject: [PATCH] video: rockchip: vpu: add VCODEC_RUNNING_MODE_HEVC for hevc

first version of rkvdec is called hevc, and it is dev_mode
is 1, add this value inside get_hw_info function

Change-Id: I0471d167029274848c00a806f0f6f0ea9f0308df
Signed-off-by: Jung Zhao <jung.zhao@rock-chips.com>
---
 drivers/video/rockchip/vcodec/vcodec_service.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/rockchip/vcodec/vcodec_service.c b/drivers/video/rockchip/vcodec/vcodec_service.c
index edd9925a8837..8314fff41fb1 100644
--- a/drivers/video/rockchip/vcodec/vcodec_service.c
+++ b/drivers/video/rockchip/vcodec/vcodec_service.c
@@ -3591,6 +3591,7 @@ static void get_hw_info(struct vpu_subdev_data *data)
 		pservice->bug_dec_addr = of_machine_is_compatible
 			("rockchip,rk30xx");
 		break;
+	case VCODEC_RUNNING_MODE_HEVC:
 	case VCODEC_RUNNING_MODE_RKVDEC:
 		pservice->auto_freq = true;
 		atomic_set(&pservice->freq_status, VPU_FREQ_BUT);
-- 
2.35.3

